docs/split_keyboards.md

```diff
@@ -76,3 +123,22 @@ In order to enable it, you must:
 - install Circuit Python in 7.2+ version
 - add `adafruit_pioasm.mpy` library to lib or root folder of a board
 - pass `use_pio=True` into `Split()` constructor
+
+The file is available [here](https://github.com/adafruit/Adafruit_CircuitPython_PIOASM/releases). From the `adafruit-circuitpython-pioasm-x.y-mpy-0.x.y.zip`, copy the `lib` folder (or its content) to the root of your circuitpython drives. Make sure to get the zip corresponding to your circuitpython version as the mpy format can evolve!
+
+Continuing our example of the KB2040, one side's UART being fliped will need to use a pin capable of only tx as rx (and vice-versa). So in order to use that qwiic/stemma connector you will also need `use_pio = True`.
+
+#### `data_pin`/`data_pin2`
+For UART `SplitType`: on the master or `split_target` side, `data_pin` is the one use for RX, `data_pin2` the one for TX.
+
+### EE HANDS
```

Comment:
Because this name is confusing for some, maybe we change this to `### EE HANDS/AUTO HANDEDNESS`